src/containers.js

```diff
@@ -1,459 +1,453 @@
 const { resetCurrentMatch, setCurrentMatch, toggleSearchInput,
-        updateSearchText } = require('./actions');
+  updateSearchText } = require('./actions');
+const { wrapperStyles, inputStyles, previousButtonStyles,
+  nextButtonStyles } = require('./containerStyles');
 const { DIRECTION_NEXT, DIRECTION_PREV, ENTER, ESCAPE } = require('./constants');
-
 exports.mapTermsState = (state, map) => (
   Object.assign(map, {
-    focussedSessionUid: state.sessions.activeUid,
-    hyperSearchToggleInput: state.ui.hyperSearchToggleInput,
-    hyperSearchInputText: state.ui.hyperSearchInputText,
-    hyperSearchLastUserSearch: state.ui.hyperSearchLastUserSearch,
-    hyperSearchCurrentRow: state.ui.hyperSearchCurrentRow,
-  })
+focussedSessionUid: state.sessions.activeUid,
```

Comment:
there are a lot of indentation changes, so this is showing a lot of changes in the code.  this makes things hard to review, can u please restore the file the to original state and only push the desired changes?
looking forward for your new code-changes, those screenshots look great!
thanks